Packages/com.quickeye.ui-toolkit-plus/Runtime/AssemblyInfo.cs

```diff
@@ -1,5 +1,7 @@
-﻿using System.Runtime.CompilerServices;
+﻿#if UNITY_EDITOR
+using System.Runtime.CompilerServices;
 using UnityEditor.UIElements;
 
 [assembly:UxmlNamespacePrefix("QuickEye.UIToolkit","quick-eye")]
 [assembly:InternalsVisibleTo("QuickEye.UIToolkit.Editor.Tests")]
+#endif
```

Comment:
Wrap only minimal necessary stuff in preprocessor directive.
```suggestion
using System.Runtime.CompilerServices;

[assembly:InternalsVisibleTo("QuickEye.UIToolkit.Editor.Tests")]

#if UNITY_EDITOR
[assembly:UnityEditor.UIElements.UxmlNamespacePrefix("QuickEye.UIToolkit","quick-eye")]
#endif
```